src/main.rs

```diff
@@ -850,8 +901,8 @@ fn main() {
         println!("'netstat' command required by ssh/smb/nfs_block failed.  Exiting.");
         std::process::exit(1);
         }
-    if config.audio_block && exist_audio().is_err() {
-        println!("'/proc/asound/' required by audio_block is unreadable.  Exiting.");
+    if config.audio_block && (exist_audio_alsa().is_err() && exist_audio_pulseaudio().is_err())  {
```

Comment:
This change isn't necessary since you made the combined `exist_audio()` function, right?  I think reverting this line is the better choice.

Reply:
I am pretty new to rust but i think i need to keep it as is. 

The idea is that `exist_audio()` just returns a boolean which will be true if either of the two methods detect audio. But if any of them fail, it should not propagate the error upwards as to not block the other detection mechanism.

So you can have either the alsa or the pulseaudio method working or even both. And the error here on startup should only be given if both methods return errors and the audio detection is enabled.

Reply:
I think that's equivalent logic to `config.audio_block && !exist_audio()`, but maybe not, and I don't care enough to find out.  This is just fine :)